src/gatling/simulations/simulations/PRL_Simulation.scala

```diff
@@ -163,11 +330,40 @@ class PRL_Simulation extends Simulation {
   }
   
   setUp(
-    PRLCitizenScenario.inject(simulationProfile(testType, prlTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption)
-  //  CafcasDownloadByDocScenario.inject(simulationProfile(testType, prlTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption)
+  //=================================================
+  //C100 & CUIRA Release Scenarios
+  //=================================================
```

Comment:
Only one scenario for master?

Reply:
Yep, need to enhance the data gen and management capabilities before being able to run all scenarios in a single test